modules/calcite/src/test/sql/join/full_outer/test_full_outer_join_many_matches.test_ignore

```diff
@@ -1,16 +1,12 @@
 # name: test/sql/join/full_outer/test_full_outer_join_many_matches.test
 # description: Test FULL OUTER JOIN with many matches
 # group: [full_outer]
-# Ignore https://issues.apache.org/jira/browse/IGNITE-15563
 
 statement ok
-PRAGMA enable_verification
+CREATE TABLE integers AS SELECT a i, 1 j FROM table(system_range(1, 2000, 1)) t1(a)
```

Comment:
To make the test work we should replace ranges to 0-1999 and 2000-3999, also we hash null values as `null`, but here assumed that null values hashed as `NULL`, so we should change hashes in this test and unignore it, or fix hashing of null values and check that other test works too.